.circleci/config.yml

```diff
@@ -958,7 +956,7 @@ workflows:
     when: << pipeline.parameters.generate_snapshots >>
     jobs:
       - run-test-ios-17:
-          xcode_version: '15.0.0'
+          xcode_version: '15.0.1'
```

Comment:
CircleCI initially shipped the update under `Xcode 15.0.0`, but now it's its own `15.0.1`